apex/contrib/csrc/multihead_attn/encdec_multihead_attn_cuda.cu

```diff
@@ -1,12 +1,15 @@
 #include <vector>
 #include <iostream>
-
+//below lines enable hip float to half conversion which are disabled by default in hip_fp16.h
+#undef __HIP_NO_HALF_OPERATORS__
+#undef __HIP_NO_HALF_CONVERSIONS__
+//#endif
```

Comment:
Not sure if these are needed.  Might be redundant.  The setup.py adds `-U` for these symbols.

Reply:
There are a few other source files where these undefs are added.  Please check whether any/all of them are needed, or if the setup.py is sufficient.

Reply:
Thanks. I will look into this.

Reply:
@hubertlu-tw I think we never addressed this redundancy? Jeff had a point that the setup.py already does the required undefines: https://github.com/ROCmSoftwarePlatform/apex/blob/62f06964d8716deb681b39ac5cc24c2cbf2d53f2/setup.py#L400